installer/tue-install-impl.bash

```diff
@@ -875,14 +903,18 @@ function tue-install-ros
     [ -n "$TUE_ROS_DISTRO" ] || tue-install-error "Environment variable 'TUE_ROS_DISTRO' is not set."
 
     local ros_pkg_name=${TUE_INSTALL_CURRENT_TARGET#ros-}
+    if [[ $ros_pkg_name == *-* ]]
+    then
+        tue-install-error "A ROS package cannot contain dashes (${ros_pkg_name})"
```

Comment:
Update this error message. 
```suggestion
        tue-install-error "A ROS package cannot contain dashes (${ros_pkg_name}). Fix target by replacing '-' with '_'."
```

Reply:
This is a misleading suggestion. People who don't know what the error is about and need this suggestion will create `ros_std_msgs` and it still won't work.

Reply:
Now provides the new name.